lib/kitchen/directions/bake_link_placeholders.rb

```diff
@@ -10,7 +10,7 @@ def self.v1(book:, cases: false)
           next unless anchor.text == '[link]'
 
           label_case = anchor['cmlnle:case'] || anchor['case']
-          id = anchor[:href][1..-1]
+          id = anchor[:href][1..]
```

Comment:
I think it does :thinking: All tests passed. I assumed that it would be visible there if id changed. Do you think we should check it in another place as well?

Reply:
@wilkus27 Ok :+1: I guess we need to trust the tests and content changes (the labels/classes are changing in the correct placesso the id matches). I should write this in the first place that I tried to google this short and couldn't found anything.